docs/fa_reference.rst

```diff
@@ -1577,6 +1648,12 @@ Smis
 .. autoclass:: pypureclient.flasharray.Smis
    :members:
 
+Smtp
+~~~~
+
+.. autoclass:: pypureclient.flasharray.Smtp
+   :members:
+
```

Comment:
Just curious, is there smtp API change in REST 2.21?

Reply:
Great point =) Fixed.